docs/source/reference/builtin-devices.rst

```diff
@@ -40,22 +40,38 @@ This must be sub-classed (like :class:`~ophyd.device.Device`) to be useful.
    ophyd.epics_motor.EpicsMotor
    ophyd.epics_motor.MotorBundle
 
+.. _built_in.Epics.Scaler.record:
```

Comment:
Is it a convention to point to the part of the documentation within the same doc (such as `#<...>` in HTML)?

Reply:
Envisioning someone who searched for one of the two scaler support classes.  The link is used to find the other (in case it was off-screen).  The NeXus manual, for example, is rife with these links.  The rendered view is not distracting.

The fiendishly-long URL to the scaler documentation is a growing pain as synApps migrates into Github-style documentation.  Original docs are in HTML.  Github shows the html source code so an additional handler (htmlpreview.github.io) is needed to render the page.

Reply:
Thanks for the explanation. I just wonder why the name is different. I thought prepending the anchor with `_` would be enough, however, that should look like `_built_in.EpicsScaler` (without the dot between `Epics` and `Scaler` and `.record`). I don't see how `built_in.Epics.Scaler.record` would be used in the references below.

Reply:
The extra anchor is not referenced as of now.  It was added to differentiate between the two sections _EPICS scaler_ and _EpicsScaler_.  By default, Sphinx creates anchors for every section based on a simple conversion of the section title to lower case and handling replacement of non-allowed characters.  The anchors in this case would be `epics_scaler` and `epicsscaler`, respectively.  These are too similar to be useful anchors.  The custom anchors avoid that ambiguity by prefixing the document name, delimiting with `.` (which is an allowed character in an anchor), and suffixing for clarity.

Since the anchor is not used, it could easily be removed.  I suggest leaving it in should a need arise in the future for a reference such as this:

    :ref:`built_in.Epics.Scaler`

Reply:
Sounds good, thanks for the explanation!